### ISSUE TITLE
Feature request: Basic documentation with  github readthedocs

### ISSUE THREAD

**Issue by scott-huberty** [lina-usc/pylossless#27]
I'm making this a separate ticket from #19 because this is more about launching an initial github `readthedocs` website for this repository. 

I want to launch a very basic skeleton of this documentation page so that we (maintainers) can see it, and iterate on it further. 

I am developing this in a branch right now. I will open a PR to merge into main (hopefully today), which will consist of the following work.

- [ ] PEP8 compliant docstring formatting in `bids.py`, `config.py` & `pipeline.py`
- [ ] sphinx files in the `docs` folder that will contain the documentation content.
- [ ] a `requirements_doc.txt` file in the project root, containing package requirements for making/contributing to the docs.

Once this PR is merged, we can generate the documentation page and I will close this ticket.

EDIT: by "generate", I mean launch the `readthedocs` page so that is accessible via URL in the web browser.
